### DISCUSSION TITLE
Passing underlying resource Id when opening inline actions

### DISCUSSION THREAD

**Discussion by KristiansJaunzems** [laravel/nova-issues#4945] [Q&A] [answered]
Could it be posible when clicking inline resource action to pass resource id to request.

![Untitled](https://user-images.githubusercontent.com/1991846/196613136-f91e08c2-26a7-499a-8d4a-8249d30b1dec.png)

In my case i would like to load default value from underlying model.

```
    public function fields(NovaRequest $request)
    {
        return [
            Number::make('Number','send')
                ->nullable()
                ->default(Model::where('id',$request->resourceId)->first()->quantity)
                ->required(),
        ];
    }
```

**Comment by sanderbaas** [comment, marked as answer]
For now maybe add a constructor to the action and pass the resource yourself as rameshguthulakkd said here: https://github.com/laravel/nova-issues/issues/2479#issuecomment-1323847264
